src/components/FormAlertWithSubmitButton.js

```diff
@@ -132,15 +69,13 @@ const FormAlertWithSubmitButton = (props) => {
                 isDisabled={props.isDisabled}
                 isLoading={props.isLoading}
             />
-        </View>
-    );
-};
+        )
+        )}
+    </FormAlertWrapper>
+);
 
 FormAlertWithSubmitButton.propTypes = propTypes;
 FormAlertWithSubmitButton.defaultProps = defaultProps;
 FormAlertWithSubmitButton.displayName = 'FormAlertWithSubmitButton';
 
-export default compose(
-    withLocalize,
-    withNetwork(),
-)(FormAlertWithSubmitButton);
+export default withLocalize(FormAlertWithSubmitButton);
```

Comment:
I can't spot where the localize stuff is used anymore. If it's not used, can you please remove this?